modules/kazoo/kz_amqp.c

```diff
@@ -684,8 +684,6 @@ int kz_amqp_add_connection(modparam_t type, void* val)
 
     if(newConn->info.vhost == NULL) {
     	newConn->info.vhost = KZ_URL_ROOT;
-    } else if(newConn->info.vhost[0] == '/' && strlen(newConn->info.vhost) == 1) { // bug in amqp_parse_url ?
-    	newConn->info.vhost++;
```

Comment:
I will rewrite this like it is done on commit https://github.com/kamailio/kamailio/commit/22e33ba64c3aad3b50e9ba649c5fa736462a30b4